stdlib/logging/config.pyi

```diff
@@ -15,9 +16,49 @@ if sys.version_info >= (3, 7):
 else:
     _Path = StrPath
 
+_Tkey = TypeVar("_Tkey", bound=Hashable)
+_TValue = TypeVar("_TValue")
+_T = TypeVar("_T", bound=Hashable)
+_V = TypeVar("_V")
+
 DEFAULT_LOGGING_CONFIG_PORT: int
 RESET_ERROR: int  # undocumented
 IDENTIFIER: Pattern[str]  # undocumented
+_listener: Optional[Any]
+dictConfigClass = DictConfigurator
+
+class ConvertingMixin(object):
+    def convert_with_key(self, key: _Tkey, value: _TValue, replace: bool) -> Dict[_Tkey, _TValue]: ...
+    def convert(self, value: _TValue) -> Dict[_Tkey, _TValue]: ...
```

Comment:
`convert` still uses a `TypeVar` that can't be determined when it's called. Maybe make `ConvertingMixin` generic, as in `class ConvertingMixin(Generic[_T, _V])`? Then you can do `class ConvertingDict(Generic[_T, _V], ConvertingMixin[_T, _V])`, for example.